.github/workflows/release.yml

```diff
@@ -40,14 +43,19 @@ jobs:
 
   release:
     name: Release
-    needs: tag
+    needs:
+      - tag
     runs-on: ubuntu-latest
+    if: ${{ needs.tag.outputs.releaseType != '' }}
     steps:
       - name: Checkout
         uses: actions/checkout@v3
         with:
           fetch-depth: 0
 
+      - name: Write Release Notes
+        run: echo "${{ needs.tag.outputs.releaseNotes }}" > releaseNotes.txt
```

Comment:
Will this file be included in the released assets?

Reply:
Yes, yes it would. Updated! Good catch!